test-case/manual/jack_detection_idle.md

```diff
@@ -0,0 +1,54 @@
+# Jack detection for headset/HDMI/DP in idle state test
+Jack detection for headset/HDMI/DP in idle state is part of a series of manual
+tests.
+
+# Preconditions
+1. Runtime PM status is suspended, no playback or capture process is running.
+
+# Test Description
+* Plug headset into unit, and determine if status is updated as expected.
+* Repeat for both HDMI and DisplayPort.
+
+## Headset
+1. Plug headset into 3 1/2 mm jack.
```

Comment:
```suggestion
1. Plug headset into 3.5 mm jack.
```

And if the test is for idle? Maybe we should check if system is in suspend first?

Reply:
> And if the test is for idle? Maybe we should check if system is in suspend first?

Thats listed as one of the preconditions of the test.